web/pgadmin/browser/server_groups/servers/databases/schemas/statistics/tests/test_statistics_get.py

```
@@ -0,0 +1,150 @@
+                # Verify that the statistics name is in the response
+                self.assertIn('name', response_data)
+                self.assertEqual(response_data['name'], self.statistics_name)
+                # Verify columns are returned
+                self.assertIn('columns', response_data)
+                # Verify stat_types are returned
+                self.assertIn('stat_types', response_data)
+        else:
+            if self.mocking_required:
+                with patch(
+                    self.mock_data["function_name"],
+                    side_effect=[eval(self.mock_data["return_value"])]
```

Ruff (0.15.15)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)